src/AssetsPage/index.jsx

```diff
@@ -56,7 +52,6 @@ AssetsPage.propTypes = {
 
 const WrappedAssetsPage = connect(
   state => ({
-    assetsList: state.assets.list,
```

Comment:
The AssetsTable was subscribed to the assetsList slice of the Redux state and was receiving it as a prop from the AssetsPage, so I've removed assetsList from here in favor of the AssetsTable subscribing to the assetsList itself.